pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -47,15 +58,14 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 	m_check_HideMouse	= new pxCheckBox( this, _("Always hide mouse cursor") );
 	m_check_CloseGS		= new pxCheckBox( this, _("Hide window when paused") );
 	m_check_Fullscreen	= new pxCheckBox( this, _("Default to fullscreen mode on open") );
-	m_check_VsyncEnable	= new pxCheckBox( this, _("Wait for Vsync on refresh") );
 	m_check_DclickFullscreen = new pxCheckBox( this, _("Double-click toggles fullscreen mode") );
 	m_check_AspectRatioSwitch = new pxCheckBox(this, _("Switch to 4:3 aspect ratio when an FMV plays"));
 	//m_check_ExclusiveFS = new pxCheckBox( this, _("Use exclusive fullscreen mode (if available)") );
 
 	m_text_Zoom->SetToolTip( pxEt( L"Zoom = 100: Fit the entire image to the window without any cropping.\nAbove/Below 100: Zoom In/Out\n0: Automatic-Zoom-In untill the black-bars are gone (Aspect ratio is kept, some of the image goes out of screen).\n  NOTE: Some games draw their own black-bars, which will not be removed with '0'.\n\nKeyboard: CTRL + NUMPAD-PLUS: Zoom-In, CTRL + NUMPAD-MINUS: Zoom-Out, CTRL + NUMPAD-*: Toggle 100/0"
 	) );
 
-	m_check_VsyncEnable->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
+	m_combo_vsync->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
```

Comment:
> I did. But I got lost once everybody in the fucking world started to have a very convinced opposite opinion

Every 1/60 s ~~ 16 ms, you have a ready interrupt from your display
* if a frame is ready, pick it up
* otherwise wait
The difference between vsync and adaptive vsync is that the former will wait the next display interrupt. Whereas the latter will wait the frame ready signal from your GPU. Adaptive vsync avoid to wait a long time when you barely miss the target. However if you're screen is busy displaying something, you will get some tearing. In short adaptive allow value between 30fps and 60fps. Whereas standard vsync is either 30fps or 60fps.

```
it seems to just be about DwmIsCompositionEnabled && window!=monitor
```
We can do something when we properly understand the situation because it is far from clear for me. And how do we test if we are in exclusive fullscreen or not.


Anyway, so far goal is to have a testbench. And then tune it. I won't merge this PR in current state.